Classes/DataProcessing/LanguageMenuProcessor.php

```diff
@@ -0,0 +1,333 @@
+                        'stdWrap.' => [
+                            'data' => 'register:languageUid'
+                        ],
+                        'wrap' => ',"languageUid":"|"'
+                    ],
+                    '71' => 'USER',
+                    '71.' => [
+                        'userFunc' => 'TYPO3\CMS\Frontend\DataProcessing\LanguageMenuProcessor->getLocalizedTitleAsJson',
+                        'languageUid.' => [
+                            'data' => 'register:languageUid'
+                        ],
+                        'stdWrap.' => [
+                            'wrap' => ',"localizedTitle":|'
+                        ]
+                    ],
+                    '72' => 'USER',
+                    '72.' => [
+                        'userFunc' => 'TYPO3\CMS\Frontend\DataProcessing\LanguageMenuProcessor->getLanguageAsJson',
+                        'languageUid.' => [
+                            'data' => 'register:languageUid'
+                        ],
+                        'stdWrap.' => [
+                            'wrap' => ',"language":|'
+                        ]
+                    ],
+                    '73' => 'USER',
+                    '73.' => [
+                        'userFunc' => 'TYPO3\CMS\Frontend\DataProcessing\LanguageMenuProcessor->getLocaleAsJson',
+                        'languageUid.' => [
+                            'data' => 'register:languageUid'
+                        ],
+                        'stdWrap.' => [
+                            'wrap' => ',"locale":|'
+                        ]
+                    ],
+                    '74' => 'USER',
+                    '74.' => [
+                        'userFunc' => 'TYPO3\CMS\Frontend\DataProcessing\LanguageMenuProcessor->getHreflangAsJson',
+                        'languageUid.' => [
+                            'data' => 'register:languageUid'
+                        ],
+                        'stdWrap.' => [
+                            'wrap' => ',"hreflang":|'
+                        ]
+                    ],
+                    '75' => 'USER',
+                    '75.' => [
+                        'userFunc' => 'TYPO3\CMS\Frontend\DataProcessing\LanguageMenuProcessor->getDirectionAsJson',
+                        'languageUid.' => [
+                            'data' => 'register:languageUid'
+                        ],
+                        'stdWrap.' => [
+                            'wrap' => ',"direction":|'
+                        ]
+                    ],
+                    '99' => 'RESTORE_REGISTER'
+                ]
+            );
+        }
+    }
+
+    /**
+     * Gets the language title for the language id submitted in $conf in JSON format
+     *
+     * @param string Empty string (no content to process)
+     * @param array TypoScript configuration
+     * @return string JSON encoded data
+     */
+    public function getTitleAsJson($content, $conf)
```

Comment:
Can we combine this and the following into a universal function?

Reply:
yes, will take a look. but what i don't like is to use $conf for that and don't see another way now. but will check again later...

Reply:
done, i now use a second parameter `field` to define the desired field in ts